src/functions.php

```
@@ -50,7 +50,7 @@
  * });
  * ```
  */
-function host(string ...$hostname): Host|ObjectProxy
+function host(string ...$hostname): any
```

PHPStan (2.1.15)

Function Deployer\host() has invalid return type Deployer\any.


(class.notFound)